just_audio/android/src/main/java/com/ryanheise/just_audio/AudioPlayer.java

```diff
@@ -80,6 +82,11 @@
     private Integer audioSessionId;
     private MediaSource mediaSource;
     private Integer currentIndex;
+    private float _speed = 1;
+    private boolean _skipSilence = false;
+    private boolean _volumeBoostEnabled = false;
+    private int _volumeBoostGainMB = 0;
```

Comment:
To follow the convention set by the rest of the file, the leading underscores should be removed.